packages/hint/docs/contributor-guide/getting-started/architecture.md

```diff
@@ -1,78 +1,81 @@
 # Architecture
 
-`webhint` was designed with extensibility in mind. At a high level, this is
-the project's architecture:
-
-[![webhint's architecture](images/architecture.svg)](images/architecture.svg)
-
-The main piece is `webhint`. It's an object that extends from `EventEmitter`.
-This means that the communication between all the different parts is going to
-be done mostly via [`event`s][events]. The other pieces are:
+The core concepts in `webhint` are:
 
 * `hint`: Is a group of related tests that are run on a resource (HTML,
-  document, image, request, etc.). E.g.: Verify that the HTML document has a
-  valid language declared.
-  [Learn how to develop a `hint`][new hint].
-* `connector`: Is the way in which `webhint` obtains information about the
-  DOM, network information, resources, etc. The underlying technique (debugging
-  protocol, web driver, etc.) to access this data does not matter to the rest
-  of the system.
-  [Learn how to develop a `connector`][new connector].
-* `formatter`: Transforms the results into something useful to the user. It
-  could be as simple as printing out the results to the command line, or
-  something more complex like creating an HTML report.
-  [Learn how to developer a `formatter`][new formatter].
+  document, image, request, etc.). E.g.: Verify that the HTML document
+  has a valid language declared. [Learn how to develop a `hint`][new
+  hint].
+* `connector`: Is the way in which `webhint` obtains information about
+  the DOM, network information, resources, etc. The underlying technique
+  (debugging protocol, web driver, etc.) to access this data does not
+  matter to the rest of the system. [Learn how to develop a
+  `connector`][new connector].
 * `parser`: Understands a particular resource type (e.g.: JavaScript,
-  stylesheet, webmanifest, etc.), parses it, and emits related event(s) so
-  `hint`s can take action on them.
-  [Learn how to develop a `parser`][new parser].
+  stylesheet, webmanifest, etc.), and exposes information about them so
+  `hint`s can take action on them. [Learn how to develop a `parser`][new
+  parser].
+* `formatter`: Transforms the results into something useful to the user.
+  It could be as simple as printing out the results to the command line,
+  or something more complex like creating an HTML report. [Learn how to
+  developer a `formatter`][new formatter].
 
-When a user invokes `hint https://example.com` basically what happens is
-this:
+All these pieces communicate with each other via [`event`s][events].
+`Engine`, which extends from `EventEmitter`, is the one enabling this
+interaction. Most of the new work is done in one of the previous
+components as `Engine` should be kept as simple as possible. The
+entities emitting events are `connector`s and `parser`s. The ones that
+subscribe to `event`s are `hint`s and `parser`s.
+
```

Comment:
Should we add a quick note about `configurations`?

Reply:
I talk a bit about `configurations` a bit later. Maybe we should have a `how to create your own configuration` section but I think that should go in another PR.

Reply:
>  in another PR.

Works for me.